### DISCUSSION TITLE
SEO pagination links is not supported in metadata options

### DISCUSSION THREAD

**Discussion by MahmoudBakr22** [vercel/next.js#49919] [App Router]
how to add SEO pagination link in metadata object 
`      <link
          rel="prev"
          key="prev"
          href={"/category?page=3"}
        />
`
I tried using the other field 
https://nextjs.org/docs/app/api-reference/functions/generate-metadata#other

but it renders it as meta tag not link tag

**Comment by devjiwonchoi** [comment]
This is very interesting. As far as I know, there is no SEO pagination link option in the Metadata options. You might wanna open an issue for this.

**Reply by MahmoudBakr22** [reply]
thanks for replaying, But I think it needs to be inside <head> tag for SEO reasons

**Reply by devjiwonchoi** [reply]
Yes, I do agree. I've looked up this issue and was not able to find a solution. I would recommend you create an issue for this, which you can do it [here](https://github.com/vercel/next.js/issues/new/choose).

**Reply by umang-goyal** [reply]
@MahmoudBakr22  @devjiwonchoi any solution or update on this issue ?

**Reply by devjiwonchoi** [reply]
@umang-goyal @MahmoudBakr22 I'll open a feature request for this, and let you know if anything comes up. For now, you could add it statically to the `<head>` tag.

```js
<html>
  <head>
    <link rel="prev" key="prev" href={someURL + '/category?page=3'} /> // here
  </head>
  <body>{children}</body>
</html>

```

**Reply by devjiwonchoi** [reply]
@MahmoudBakr22 @umang-goyal I've done some research and seems like this pagination method for SEO is pretty much outdated.

I've added feature request #52926. Let's see how it goes.

### Reference:
- [Pagination with rel="next" and rel="prev"](https://developers.google.com/search/blog/2011/09/pagination-with-relnext-and-relprev)
- [Pagination, incremental page loading, and their impact on Google Search](https://developers.google.com/search/docs/specialty/ecommerce/pagination-and-incremental-page-loading-)

**Reply by devjiwonchoi** [reply]
@MahmoudBakr22 @umang-goyal  Alternate for now:

```ts
export const metadata = {
  other: [
    {
      rel: 'prev',
      href: 'someURL',
    },
    {
      rel: 'next',
      href: 'someURL',
    },
  ],
}
```

**Comment by devjiwonchoi** [comment]
No further updates, my PR is not being accepted for a long time, so let's stick to the alternate way for now.

```ts
export const metadata = {
  other: [
    {
      rel: 'prev',
      href: 'https://example.com/pagination/1',
    },
    {
      rel: 'next',
      href: 'https://example.com/pagination/3',
    },
  ],
}
```

**Reply by defx** [reply]
This solution doesn't work for me, which version are you using? According to the docs, `metadata.other` only supports the creation of meta tags, not links.
